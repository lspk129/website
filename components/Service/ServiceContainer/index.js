import styled from 'styled-components';

const ServiceContainer = styled.div`
  text-align: center;
  display: grid;
  grid-template-columns: 375px 375px;
  grid-template-rows: 260px;
  padding-bottom: 100px;
  justify-content: center;
  align-content: center;
  
`;

export default ServiceContainer;
