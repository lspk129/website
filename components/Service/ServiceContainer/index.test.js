import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';

import ServiceContainer from './index';

describe('ServiceLayout', () => {
  it('renders corretcly', () => {
    const wrapper = renderer.create(<ServiceContainer />);
    const tree = wrapper.toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('renders div element', () => {
    const wrapper = shallow(<ServiceContainer />);
    expect(wrapper).toBePresent();
    expect(wrapper).toHaveTagName('div');
  });
});
