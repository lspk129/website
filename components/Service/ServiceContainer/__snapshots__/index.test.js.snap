// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ServiceLayout renders corretcly 1`] = `
<div
  className="sc-bdVaJa jFsOXn"
/>
`;
